README.md

```diff
@@ -23,8 +23,9 @@ The state in which the Postfix service should be after this role runs, and wheth
 
     postfix_inet_interfaces: localhost
     postfix_inet_protocols: all
+    postfix_mynetworks: 127.0.0.0/8 [::ffff:127.0.0.0]/104 [::1]/128 
```

Comment:
Isn't the default to not specify it at all (and allow forwarding to any network)? If so, I'd like the role default to be `''` and if it's not set, then it would be omitted (`| default(omit)`, something like that).

Reply:
Changed default behavior to omit.